Kitodo-API/src/main/java/org/kitodo/config/Config.java

```diff
@@ -11,141 +11,197 @@
 
 package org.kitodo.config;
 
+import java.net.URI;
+import java.nio.file.Paths;
 import java.util.NoSuchElementException;
+import java.util.Optional;
 
+import org.apache.commons.configuration.AbstractConfiguration;
 import org.apache.commons.configuration.ConfigurationException;
 import org.apache.commons.configuration.ConversionException;
 import org.apache.commons.configuration.PropertiesConfiguration;
 import org.apache.commons.configuration.reloading.FileChangedReloadingStrategy;
+import org.apache.commons.io.FilenameUtils;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
 public class Config {
     private static final Logger logger = LogManager.getLogger(Config.class);
     private static volatile PropertiesConfiguration config;
     private static final String CONFIG_FILE = "kitodo_config.properties";
-    private static final String METADATA_DIRECTORY = "MetadatenVerzeichnis";
-    private static final String CONFIG_DIR = "KonfigurationVerzeichnis";
+    public static final int INT_PARAMETER_NOT_DEFINED_OR_ERRONEOUS = 0;
+
+    public static final String DIR_MODULES = "moduleFolder";
```

Comment:
Some variables are marked as public other as private. Also necessary to decide for one convention.

Reply:
`DIR_PROCESSES` and `DIR_XML_CONFIG` have corresponding getter function, `getKitodoDataDirectory()` and `getKitodoConfigDirectory()`. `DIR_MODULES` is used as constant form `KitodoServiceLoader` and must therefore be public.

Reply:
Ok.